### PR TITLE
Deepgram .NET SDK v3.0.0

### PR DESCRIPTION
**Pull request by lukeocodes**

### CODE REVIEW

File: SampleApp/.cr/personal/FavoritesList/List.xml
Comment (ThindalTV) [inline SampleApp/.cr/personal/FavoritesList/List.xml:1]: In general, you don't want to add the coderush config files to the repo. Add .cr\* to the gitignore file.

File: Deepgram.Tests/Deepgram.Tests.csproj [hunk 1]
```diff
@@ -1,28 +1,27 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
-  <PropertyGroup>
-    <TargetFramework>net6.0</TargetFramework>
-    <ImplicitUsings>enable</ImplicitUsings>
-    <Nullable>enable</Nullable>
+	<PropertyGroup>
+		<TargetFrameworks>net7.0;netstandard2.0</TargetFrameworks>
```
Comment (CopperBeardy) [inline Deepgram.Tests/Deepgram.Tests.csproj:4]: remove ;netstandard2.0

File: SampleApp/.cr/personal/FavoritesList/List.xml
Comment (CopperBeardy) [reply SampleApp/.cr/personal/FavoritesList/List.xml:1]: yeah keep meaning to do that

**Review by CopperBeardy — COMMENTED** [review]
All seems good

**Comment by lukeocodes** [comment]
Ready to MERGE

**Review by SandraRodgers — APPROVED** [review]

**Pull request merged**